package.json

```diff
@@ -37,7 +37,6 @@
     "jsdoc-to-markdown": "^2.0.1",
     "mocha": "^3.2.0",
     "mochainon": "^1.0.0",
-    "wary": "^1.1.0"
```

Comment:
No, but we don't need that in this case, as the output shouldn't change anyways, and if it does, we'd want to be able to inspect it (which is hard to do with wary).